apps/api/prisma/schema.prisma

```diff
@@ -54,6 +59,60 @@ model PurchaseOrders {
   updated_at             DateTime?
 
   purchase_order_line_items PurchaseOrderLineItems[]
+  good_receipts GoodsReceipt[]
+  supplier_invoices SupplierInvoice[]
 
   @@map("purchase_orders")
 }
+
+model GoodsReceipt {
+  id                Int                 @id @default(autoincrement())
+  purchase_order    PurchaseOrders      @relation(fields: [purchase_order_id], references: [id])
+  purchase_order_id Int
+  received_date     DateTime
```

Comment:
following pattern of created_at, updated_at, received_at could be an idea here

Reply:
Good catch. Done!